src/components/Switch/Switch.tsx

```diff
@@ -7,9 +9,15 @@ function Switch(props: ISwitch) {
     props.setChecked(ev.target.checked);
   };
 
+  const hasErro = true;
```

Comment:
Esse cara pode ser mais fiel ao que deveria ser o final, talvez a implementação de um state simples ao invés de uma const chumbada?

Also o nome correto é hasError
```suggestion
  const hasError = true;
```